domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/service/UserRoleServiceImplTest.java

```diff
@@ -10,29 +11,36 @@
 import uk.gov.hmcts.ccd.definition.store.repository.model.UserRole;
 
 import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
 import java.util.Optional;
 
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
 import static org.mockito.BDDMockito.given;
 import static org.mockito.Mockito.*;
+import static uk.gov.hmcts.ccd.definition.store.repository.SecurityClassification.PUBLIC;
+import static uk.gov.hmcts.ccd.definition.store.repository.SecurityClassification.RESTRICTED;
 
-public class UserRoleServiceImplTest {
+class UserRoleServiceImplTest {
 
     private UserRoleRepository repository;
     private UserRoleService service;
 
     private UserRoleEntity entity = mock(UserRoleEntity.class);
     private UserRole mockUserRole = mock(UserRole.class);
 
-    @Before
-    public void setUp() {
+    @BeforeEach
+    void setUp() {
         repository = mock(UserRoleRepository.class);
         service = new UserRoleServiceImpl(repository);
     }
 
     @Test
-    public void getRole_shouldGetRole() {
+    @DisplayName("should return role if defined")
+    void getRole_shouldGetRole() {
```

Comment:
I would suggest using `@Nested` here to group tests and remove the need for a `getRole` prefix.

Reply:
👍